data_providers/event_provider.gd

```diff
@@ -0,0 +1,94 @@
+class_name EventProvider extends JSONLoader
+
+var file_path = "res://data_providers/data/events.json"
+
+var events
+
+# Es werden von den Get - Funktionen immer 
+# 		[0] ein String (der Text zum Event) und
+# 		[1] ein Wert (im Zusammenhang zum Event) 
+# 	als liste zurück gegeben
```

Comment:
Do not use any arry for this but its own event class. As a rule of thumb, if a caller has to look up what your function is returning and cannot guess it by the name, it sounds like a code smell.

_Also, the code language is English._